Makefile

```diff
@@ -1,8 +1,8 @@
-## This Makefile is meant to be run on Unix hosts - as such, it only supports the few
-## operations that don't require on a Windows host, mostly code generation and linting.
+os=windows
+CMDS=server
```

Comment:
"server" seems too generic of a binary name. Should it be called something more specific like "csi-proxy-server"?

Reply:
I will update the dir name in a separate PR to reduce the changes in this one.